Disable input prompts with fixed settings for installation of MIME-Lite
author: Alex Domingo (Vrije Universiteit Brussel)
--- Makefile.PL.orig	2024-12-21 11:29:33.708553000 +0100
--- Makefile.PL.orig	2024-12-21 16:34:19.324679196 +0100
+++ Makefile.PL	2024-12-21 16:34:58.365068363 +0100
@@ -33,10 +33,7 @@
         (map {; "\t$_\n" } sort keys %extra_prereq),
         "I can add these modules to the prereq list which will cause the files to be\n",
         "automatically installed if they aren't already present\n";
-
-  if (prompt("Add prereqs?",'Yes')=~/y/i) {
-    %prereq = (%prereq, %extra_prereq);
-  }
+  %prereq = (%prereq, %extra_prereq);
 }
 
 # Write the Makefile:
